nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -38,3 +38,9 @@ In addition to numerous new and upgraded packages, this release has the followin
 - The module for the application firewall `opensnitch` got the ability to configure rules. Available as [services.opensnitch.rules](#opt-services.opensnitch.rules)
 
 - A new `virtualisation.rosetta` module was added to allow running `x86_64` binaries through [Rosetta](https://developer.apple.com/documentation/apple-silicon/about-the-rosetta-translation-environment) inside virtualised NixOS guests on Apple silicon. This feature works by default with the [UTM](https://docs.getutm.app/) virtualisation [package](https://search.nixos.org/packages?channel=unstable&show=utm&from=0&size=1&sort=relevance&type=packages&query=utm).
+
+- The module `services.headscale` was refactored to be compliant with [RFC 0042](https://github.com/NixOS/rfcs/blob/master/rfcs/0042-config-option.md). To be precise, this means that the following things have changed:
+  - Most settings has been migrated under [](#opt-services.grafana.settings) which is an attribute-set that
```

Comment:
`[]` probably needs to be filled & the link updated to not be `grafana`

Reply:
This list is pretty short so it's bound to happen right now but the advice is to put the item in at a random point in this list to avoid merge conflicts

Reply:
Addressed